Java/JDI/jdi-mock-rest/src/main/java/com/epam/mock/service/ConfigurationHandlerService.java

```diff
@@ -0,0 +1,98 @@
+package com.epam.mock.service;
+
+import com.epam.mock.model.ConfigurationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.stereotype.Service;
+
+/**
+ * Response service class.
+ */
+@Service
+public class ConfigurationHandlerService {
+
+    private static Map<String, ConfigurationHandler> configurationHandlerMap = new HashMap<>();
+
+    static {
+        configurationHandlerMap.put("getusers", new ConfigurationHandler("getusers", 200, "{\"name\":\"roman\", \"id\":\"300\",\"role\":\"admin\"}"));
+        configurationHandlerMap.put("getuser", new ConfigurationHandler("getuser", 200, "{\"name\":\"roman\", \"id\":\"300\",\"role\":\"admin\"}"));
+        configurationHandlerMap.put("registeruser", new ConfigurationHandler("registeruser", 404, ""));
+        configurationHandlerMap.put("updateuser", new ConfigurationHandler("updateuser", 200, ""));
+        configurationHandlerMap.put("updateuserfield", new ConfigurationHandler("updateuserfield", 200, ""));
+        configurationHandlerMap.put("clearuser", new ConfigurationHandler("clearuser", 404, ""));
+        configurationHandlerMap.put("clearall", new ConfigurationHandler("clearall", 200, ""));
+    }
+
+    /**
+     * Responses to controller call for all users.
+     * @return Our representation of response with JSON format.
+     */
+    public ConfigurationHandler getUsers() {
+        return configurationHandlerMap.get("getusers");
```

Comment:
"getusers" - extract to constant at the beginning of the class ... or even introduce Enum with applicable methods. Here and in all further methods in this class.